code/modules/mob/living/carbon/xenomorph/castes/spiderling/spiderling.dm

```diff
@@ -128,24 +135,26 @@
 
 /// This happens when the spiderlings mother dies, they move faster and will attack any nearby marines
 /datum/ai_behavior/spiderling/proc/spiderling_rage()
+	SIGNAL_HANDLER
 	var/mob/living/carbon/xenomorph/spiderling/x = mob_parent
 	var/list/mob/living/carbon/human/possible_victims = list()
-	for(var/mob/living/carbon/human/victim in cheap_get_humans_near(x, SPIDERLING_RAGE_RANGE))
+	for(var/mob/living/victim in get_nearest_target(x, SPIDERLING_RAGE_RANGE))
 		if(victim.stat == DEAD)
 			continue
 		possible_victims += victim
 	if(!length(possible_victims))
 		kill_parent()
 		return
-	x.emote("roar")
+	// Makes the spiderlings roar at slightly different times so they don't stack their roars
+	addtimer(CALLBACK(x, TYPE_PROC_REF(/mob, emote), "roar"), rand(50, 200)  MILLISECONDS)
```

Comment:
Also very ew

Reply:
Do you have suggestions on how to randomize the timings in a better way? Otherwise they all roar/slash at the same time, and it's not great.